### ISSUE TITLE
Set a max line length

### ISSUE THREAD

**Issue by jakubfiala** [wellcometrust/eslint-config-wellcome#1]
Not sure what everyone's opinion on this is, but I think it may be useful to set a reasonable max line length. Long lines are ugly.

AirBnB uses a 100 char max length, which I think is fine, so we could start with that …

@wellcometrust/developers

**Comment by jamesgorrie** [comment]
I use `100` in my IDE (but don't always adhere to it) - so I'd be happy with 100.

**Comment by jakubfiala** [comment]
yeah I think there are cases where it's justified, e.g. if there's a long-ass base64 string. But having it as a lint rule won't hurt :)

**Comment by jamesgorrie** [comment]
> won't hurt

Unless it slows things down, but this again is my bias towards not linting.

**Comment by davidpmccormick** [comment]
I think I'm OK with it. I used to use AirBnB's config and I didn't have to fight against it. Much.

**Comment by jakubfiala** [comment]
kewl, seems like there's largely a consensus about this, I'll make a PR.
